### PR TITLE
Update setup w prisma and fetch-shim

### PR DESCRIPTION
**Pull request by exrhizo**
Fix the build instructions, running `prisma generate` is needed to create prisma's node_modules

Also, the global fetch to be included with an import
Having it in the generated files is hard to maintain.

Rename `setup-hardhat.sh` to `setup.sh`

### CODE REVIEW

File: scripts/generate-zeus.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ function generate() {
   local MV_PATH=$GEN_PATH/$1; shift
 
   # pass the rest of the arguments to zeus
-  zeus $HASURA_GRAPHQL_ENDPOINT $GEN_PATH --ts --rq $@
+  (set -x; zeus $HASURA_GRAPHQL_ENDPOINT/v1/graphql $GEN_PATH --ts --rq $@)
```
Comment (levity) [inline scripts/generate-zeus.sh:15]: Why did this change?

**Review by levity — DISMISSED** [review]

File: scripts/generate-zeus.sh (hunk 1 above)
Comment (exrhizo) [reply scripts/generate-zeus.sh:15]: because it was wrong, from when I wrote it

**Comment by exrhizo** [comment]
@levity 
 
Maybe I should just rename `setup.sh` back but this vercel command doesn't live in the repo?
 
 
image

**Comment by levity** [comment]
>  @levity 
> 
>  
> 
> Maybe I should just rename `setup.sh` back but this vercel command doesn't live in the repo?

Right, either rename back or change the Vercel config

File: scripts/generate-zeus.sh (hunk 1 above)
Comment (levity) [reply scripts/generate-zeus.sh:15]: that's weird, it was working for me as it was--i ran this several times to test it when i refactored it

File: api-lib/node-fetch-shim.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import fetch from 'node-fetch';
+
+// This injects fetch into the global context for node.
+if (!globalThis.fetch) {
+  globalThis.fetch = fetch as unknown as typeof globalThis.fetch;
```
Comment (levity) [inline api-lib/node-fetch-shim.ts:5]: do we still need the `@types/node-fetch` changes from ba558f9c? if not, can they be removed?

Comment (exrhizo) [reply api-lib/node-fetch-shim.ts:5]: That was done wrong, I made a mistake merging it, @jsur put that in the generated code, if we want it in a place like that it should be in a place like `api-lib/Gql.ts` but I want to refactor all of that

File: scripts/generate-zeus.sh (hunk 1 above)
Comment (exrhizo) [reply scripts/generate-zeus.sh:15]: weird, maybe it was something with renaming variables actually

File: api-lib/node-fetch-shim.ts (hunk 2 above)
Comment (levity) [reply api-lib/node-fetch-shim.ts:5]: sorry, i should have been more specific i guess. i meant specifically adding `@types/node-fetch` to `devDependencies` in package.json. if that's not needed, let's remove it

File: scripts/generate-zeus.sh (hunk 1 above)
Comment (levity) [reply scripts/generate-zeus.sh:15]: hmm. ideally we wouldn't leave this `/v1/graphql` bit here when we have it in the env variable in other cases, for consistency's sake

**Pull request merged**